README.md

```diff
@@ -211,7 +211,7 @@ Feel free to check the [issues page]( https://github.com/juaryimami/OOP_SCHOOL_L
 
 > Write a message to encourage readers to support your project
```

Comment:
- [ ] Kindly remove these sections that provide instructions on how you are supposed to customize your readme as they are only supposed to guide you. Also, remember that the `FAQ` section is optional. You can fill it or remove it